.eslintrc.json

```diff
@@ -1,11 +1,8 @@
 {
   "root": true,
   "parser": "@typescript-eslint/parser",
-  "plugins": ["@18f/eslint-plugin-identity", "@typescript-eslint"],
-  "extends": [
-    "plugin:@18f/eslint-plugin-identity/recommended",
-    "plugin:@typescript-eslint/recommended"
-  ],
+  "extends": ["plugin:@18f/eslint-plugin-identity/recommended"],
+  "plugins": ["@18f/eslint-plugin-identity"],
```

Comment:
Can we now remove any / all of the explicit `rules` defined below?

Reply:
Looks like we can shrink to at least this:

```json
{
  "root": true,
  "extends": ["plugin:@18f/eslint-plugin-identity/recommended"],
  "plugins": ["@18f/eslint-plugin-identity"],
  "parserOptions": {
    "sourceType": "module",
    "ecmaVersion": 2021
  },
  "env": {
    "node": true,
    "browser": true
  },
  "rules": {
    "jsx-a11y/label-has-associated-control": ["error", { "assert": "either" }],
    "no-shadow": "off",
    "import/no-extraneous-dependencies": ["error", { "devDependencies": ["**/*.test.*", "./*"] }],
    "react/no-unknown-property": [
      "error",
      { "ignore": ["font-size", "stroke-width", "text-anchor"] }
    ]
  },
  "settings": {
    "react": {
      "pragma": "h",
      "fragment": "Fragment",
      "version": "16.2"
    }
  },
  "ignorePatterns": ["_site/**"]
}
```

Reply:
love it, 6ad382d7d95ae7e7d917cf05b570d54e7fc53c89